CRM/Cdntaxcalculator/BAO/CDNTaxes.php

```diff
@@ -20,30 +22,36 @@ static public function applyTaxesToPriceset(&$feeBlock, &$taxes) {
           $has_taxable_amounts = TRUE;
         }
       }
+
+      CRM_Utils_Hook::singleton()->invoke(['line_items'], $fee['options'], $null, $null, $null, $null, $null, 'cdntaxcalculator_alter_lineitems');
```

Comment:
This is not a standard way to define hooks or invoke hook calls.  CRM_Utils_Hook_CDN Create a class that will extend CRM_Utils_Hook and then define hook and invoke  the hook call here CRM_Utils_Hook_CDN::alterLineItems();